style/_game.sass

```diff
@@ -0,0 +1,171 @@
+.main__game
+    margin-top: 50px
+    font-size: 18px
+    display: flex
+    flex-direction: column
+    align-content: center
+    justify-content: center
+    align-items: center
+
+.game__selected-item
+    padding: 10px 20px 10px 20px
+    display: flex
+    flex-direction: row
+    align-content: center
+    justify-content: center
+    align-items: center
+
+    .selected
+        display: flex
+        flex-direction: row
+        align-content: center
+        justify-content: center
+        align-items: center
+        column-gap: 20px
+
+        &__button-cross:hover,
+        &__button-circle:hover
+            transform: scale(1.1)         
+            transition: transform 300ms ease-out
+.cross
+    width: 50px
+    height: 5px
+    padding: 15px 0px 15px 0px
+    z-index: -1
+
+.cross:before,
+.cross:after
+    position: absolute
+    content: ""
+    width: 50px
+    height: 5px
+    background-color: #000
+
+.cross:hover:before,
+.cross:hover:after
+    background-color: #fff
+    transition: background-color 500ms ease-out
+    
+.cross:before 
+    transform: rotate(45deg)
+  
+.cross:after
+    transform: rotate(-45deg)
+
+.circle
+    width: 35px
+    height: 35px
+    border: 5px solid #000
+    border-radius: 50%
+    z-index: -1
+
+.circle:hover
+    border-color: #fff
+    transition: all 500ms ease-out
+
+.selection:before,
+.selection:after
+    background-color: #fff
+
+.selection
+    border-color: #fff
+
+.game__grid
+    margin-top: 50px
+    display: flex
+    align-items: center
+    justify-content: center
+    flex-direction: column
+
+.grid
+    width: 250px
+    height: 250px
+    display: flex
+    flex-flow: row wrap
+    align-content: center
+    justify-content: center
+    align-items: center
+   
+    &__box
+        width: 70px
+        height: 70px
+        border: 1px solid #000
+        display: flex
+        align-items: center
+        justify-content: center
+        position: relative   
+    
+    &__box:first-child,
+    &__box:nth-child(2),
+    &__box:nth-child(3)
+        border-top: none
+
+    &__box:nth-child(3n + 1)
+        border-left: none
+
+    &__box:nth-child(3n + 3)
+        border-right: none
+
+    &__box:last-child,
+    &__box:nth-child(8),
+    &__box:nth-child(7)
+        border-bottom: none
+
+.player,
+.computer
+    pointer-events: none
+
+.disable
+    opacity: 0.5
+    color: gray
+    transition: all 1s ease
+
+.line-horizontal
+    width: 0px
+    height: 5px
+    background: black
+    position: absolute
+    left: 0
+    animation: width 500ms linear both
+
+.line-vertical
+    width: 5px
+    height: 0px
+    background: black
+    position: absolute
+    top: 0
+    animation: height 500ms linear both
+    transform: rotate(180deg)
+
+.svg-cross-wise
+    width: 400px
+    height: 400px
+    position: absolute
+
+.cross-wise
+    stroke: #000
```

Comment:
it would be nice to move all colors to the separate file like `variables.sass` or `palette.sass`